### PR TITLE
Feature/254 email reply scroll

### PR DESCRIPTION
**Pull request by ruilealsilva**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -50,5 +50,17 @@
     "jest-date-mock": "^1.0.3",
     "prettier": "^1.13.5",
     "redux-mock-store": "^1.5.3"
+  },
+  "browserslist": {
```
Comment (bendiggle) [inline frontend/package.json]: Any reason why you added this? We can't guarantee what browser the users are going to use, so best not to have it

File: frontend/src/components/ActionLink/ActionLinkwithClick.js [hunk 2]
```diff
@@ -68,6 +70,7 @@ function ActionLinkwithClick({
           timestamp: new Date(),
         });
         remove && markThreadAsDeleted(threadId) && markThreadAsInactive();
+        onReplyPress();
```
Comment (bendiggle) [inline frontend/src/components/ActionLink/ActionLinkwithClick.js]: You will need an if statement here to check if there is an `onReplyPress` function` as the behaviour is a bit weird for the other buttons that aren't reply.
```suggestion
        if (ononReplyPress) onReplyPress();
```

File: frontend/package.json (hunk 1 above)
Comment (ruilealsilva) [reply frontend/package.json]: When we updated react scripts, it asks to add browserslist.

File: frontend/src/components/ActionLink/ActionLinkwithClick.js (hunk 2 above)
Comment (ruilealsilva) [reply frontend/src/components/ActionLink/ActionLinkwithClick.js]: Done.

File: frontend/package.json (hunk 1 above)
Comment (bendiggle) [reply frontend/package.json]: Ah yeah that's annoying! We do this on the platform so try this `"browserslist": "last 2 versions, ie 10"`

**Pull request merged**